issue_template.md

```diff
@@ -0,0 +1,14 @@
+# How to contribute
+
+In order to help diagnose and fix your issue we often need your "debug information". It's easy to save this information and attach it to a GitHub issue:
+
+  * Start UMS.
+  * Go to the `Logs` tab.
+  * Press the `Create TRACE logs` button which restarts UMS in the "TRACE" mode. If that doesn't work for some reason, either select `Trace` from the `Log Level` dropdown at the bottom or set `log_level = TRACE` in the configuration file and restart UMS manually.
+  * Reproduce the problem.
+  * Click `Pack debug files` on the lower left.
+  * Click `Zip selected files`.
+  * Save the zip file to a location you will remember.
+  * Attach the zip file to the GitHub issue.
```

Comment:
Sadly nothing is said about the size limit...
It should also have make some sens to first update _CONTRIBUTING.md_, imho...

Reply:
@Sami32 good point. I see that we made something what is already there.